src/lang/std/artifactGraph.ts

```diff
@@ -436,10 +436,10 @@ export function getArtifactsToUpdate({
       response.data.modeling_response.type === 'solid3d_get_opposite_edge' &&
       response.data.modeling_response.data.edge) ||
     // or is adjacent edge
-    (cmd.type === 'solid3d_get_prev_adjacent_edge' &&
+    (cmd.type === 'solid3d_get_next_adjacent_edge' &&
```

Comment:
I believe this is related to https://github.com/KittyCAD/engine/issues/2467 and the fix for it engine side, writing the tests for sketch on chamfer I a bug and this is the fix (along with a rust change)